_quarto.yml

```diff
@@ -0,0 +1,60 @@
+project:
+  type: website
+
+website:
+  title: REPO_NAME
+  favicon: android-chrome-512x512.png
+  open-graph: true
+  twitter-card: true
+  site-url: https://USERNAME.github.io/REPO_NAME/
+  repo-url: https://github.com/USERNAME/REPO_NAME/
+  issue-url: https://github.com/USERNAME/REPO_NAME/issues/new/choose
+  repo-actions: [edit, issue]
+  page-navigation: true
+  bread-crumbs: true
+  back-to-top-navigation: true
+  search:
+    show-item-context: true
+    type: overlay
+  navbar:
+    logo: android-chrome-512x512.png
+    logo-alt: ''
+    title: REPO_NAME
+    tools:
+      - icon: github
+        url: https://github.com/USERNAME/REPO_NAME/
+    left:
+      - text: Readme
+        href: index.qmd
+      - text: Report
+        href: report.md
+
+  sidebar:
+    contents:
+      - section: REPO_NAME
+        href: index.qmd
+        contents:
+          - text: Changelog
+            href: CHANGELOG.md
+          - text: Code of Conduct
+            href: CODE_OF_CONDUCT.md
+          - text: Contributing
+            href: CONTRIBUTING.md
+          - text: License
+            href: LICENSE.md
```

Comment:
This won't work with #14 since `LICENSE.md` is removed

- add two license links (data + code),
- change `href: LICENSE.md` to `href: LICENSE-CCBY` or
- restore `LICENSE.md` and have an overview page over licensing details, linking to `LICENSE-CCBY` and `LICENSE-AGPL`?

Reply:
Thanks for the review. I added two different links.